libcxx/test/std/algorithms/alg.modifying.operations/alg.fill/fill_n.pass.cpp

```diff
@@ -15,159 +15,153 @@
 
 #include <algorithm>
```

Comment:
Could you format this in a separate patch? It's really not obvious what you've changed.

Reply:
Okay sure.